cypress/integration/login.spec.js

```diff
@@ -0,0 +1,18 @@
+import { createPublicKey } from "crypto";
+
+const { cy } = global;
+
+describe('login', () => {
+	it('should be able to login a user', () => {
+    cy.login();
+    cy.server({urlMatchingOptions: { matchBase: false, dot: true }});
+    cy.route('GET', '/users/current', 'fixture:users/current/get');
+    cy.route('GET', '/mentors?limit=*', 'fixture:mentors/get');
+    cy.visit('/')
+    cy.getByTestId('user-avatar')
+      .getByAltText('brentmclark@gmail.com')
```

Comment:
Is it necessary anymore?

Reply:
Technically not, no; I can remove it if you'd prefer.  

If it helps, I decided to keep the assertion for the alt text in place – as well as the other assertions – after thinking through exactly what would give us the most confidence that the app is responding properly to the authenticated state.  To me, confidence was achieved after the following:

- An element with the `testid` of `user-avatar`
- An image containing the user's avatar to act as the clickable area for the user menu dropdown
- A link inside the user menu dropdown with the text `Logout`

Does this resonate with you, or are there more/fewer items required to achieve confidence here?

Reply:
The checks are perfect. I just didn't notice that this line is an assertion, because I use to see assertion as something like "except something to be some value".

Reply:
ohhh, I see where you're coming from.  Yep; that was an odd one for me as well when I first started using the [commands we've added](https://github.com/Coding-Coach/find-a-mentor/blob/master/cypress/support/index.js#L2) from [`@testing-library/react`](https://testing-library.com/docs/dom-testing-library/api-queries#getby).  The `getBy` operations *are* assertions; in fact they are two assertions at the same time.  When you use a `getBy` operation you're asserting that the element is present, *and* that there is only 1.

Reply:
Interesting 🕵️‍♂️